rmc/resources/grch37/rmc.py

```diff
@@ -0,0 +1,374 @@
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing transcripts with no significant breaks.
+"""
+
+rmc_results = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{CONSTRAINT_PREFIX}/{CURRENT_VERSION}/{freeze}/all_rmc.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing all transcripts with evidence of regional missense constraint.
+
+Contains transcripts with one or additional breaks plus simultaneous breaks results.
+"""
+
+context_with_oe = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{CONSTRAINT_PREFIX}/{CURRENT_VERSION}/{freeze}/context_with_oe.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing missense variants in canonical transcripts annotated with missense OE.
+
+Each row is annotated with RMC transcript subsection missense OE if it exists, otherwise
+the transcript level missense OE.
+"""
+
+context_with_oe_dedup = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{CONSTRAINT_PREFIX}/{CURRENT_VERSION}/{freeze}/context_with_oe_dedup.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Deduplicated version of `context_with_oe`.
+
+Some locus/alleles combinations are present more than once in `context_with_oe` if they have
+a most severe consequence of 'missense_variant' in more than one canonical transcript.
+
+This Table contains only one row per each unique locus/alleles combination.
+"""
+
+rmc_browser = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{CONSTRAINT_PREFIX}/{CURRENT_VERSION}/{freeze}/rmc_browser.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing all transcripts with evidence of regional missense constraint.
+
+Contains same information as `rmc_results` but has different formatting for gnomAD browser.
+"""
+
+# Missense badness related resources
+amino_acids_oe = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{MPC_PREFIX}/{CURRENT_VERSION}/{freeze}/amino_acid_oe.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing all possible amino acid substitutions and their missense OE ratio.
+
+Input to missense badness calculations.
+"""
+
+misbad = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{MPC_PREFIX}/{CURRENT_VERSION}/{freeze}/missense_badness.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing all possible amino acid substitutions and their missense badness scores.
+"""
+
+joint_clinvar_gnomad = VersionedTableResource(
```

Comment:
This resource should go under the MPC resources section rather than missense badness.